flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/StreamTaskNetworkInput.java

```diff
@@ -148,6 +148,9 @@ public boolean isFinished() {
 
 	@Override
 	public CompletableFuture<?> isAvailable() {
+		if (currentRecordDeserializer != null) {
```

Comment:
Nit: I would use ternary operator instead of `if` here

Reply:
Personally I'm not a fan of them, I hope that won't be a review blocker ;)